docs/man/docker.1.md

```diff
@@ -53,6 +53,9 @@ To see the man page for a command run **man docker <command>**.
 **-e**, **--exec-driver**=""
   Force Docker to use specific exec driver. Default is `native`.
 
+**--exec-opt**=[]
+  Set exec driver options. See EXEC DRIVER OPTIONS.
+
```

Comment:
Looks like an unrelated change. (I like it, but don't think it belongs in this PR)